app/Http/Middleware/CheckFeature.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Ushahidi\App\Http\Middleware;
+
+use Ushahidi\App\Facades\Features;
+use Closure;
+
+class CheckFeature
+{
+	/**
+	 * Handle an incoming request.
+	 *
+	 * @param  \Illuminate\Http\Request  $request
+	 * @param  \Closure  $next
+	 * @param  mixed  ...$features
+	 * @return mixed
+	 */
+	public function handle($request, Closure $next, ...$features)
+	{
+		foreach ($features as $feature) {
+			if (!Features::isEnabled($feature)) {
```

Comment:
TBH I actually preferred this being a simple if statement within routes. My thinking was that feature toggle shouldn't be complicated, and also that routes simply shouldn't exist if the feature is disabled (vs existing but having middleware). 

That said: I'm not strongly attached to my approach. If you/@rowasc prefer this way, then lets do it. As long as we pick a method and stick with it.